### PR TITLE
fix: Abscence of a logo on the Sign In and Sign Up pages gf-121

### PR DESCRIPTION
**Pull request by kolibri753**
The parent component (Auth) is responsible for deciding whether the image has loaded or failed, by passing down the isImageLoaded state and onImageError handler as props to the Logo component. The Logo component then uses these props to conditionally render either the image or a fallback (styled "LOGO" text).

image

### CODE REVIEW

File: apps/frontend/src/pages/auth/auth.tsx [hunk 1]
```diff
@@ -59,12 +61,20 @@ const Auth = (): JSX.Element => {
...
 			<section className={styles["auth-container"]}>
 				<div className={styles["left-side"]}>
-					{/* TODO: replace logo */}
-					<img alt="logo" className={styles["logo-wrapper"]} src="" />
+					{/* TODO: add logoSrc */}
+					<Logo
+						isImageLoaded={isImageLoaded}
+						logoSrc=""
+						onImageError={handleImageError}
+					/>
```
Comment (liza-veis) [inline apps/frontend/src/pages/auth/auth.tsx:77]: Too complicated logic for this case 😅 let's keep it as it was, the only thing that is needed is to use a random image as a placeholder. You can download "LOGO" from the design as an image for this.

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/auth/auth.tsx (hunk 1 above)
Comment (liza-veis) [reply apps/frontend/src/pages/auth/auth.tsx:77]: I think we don't need to handle case when image is not loaded, the application won't build if we we will import invalid image path

Comment (kolibri753) [reply apps/frontend/src/pages/auth/auth.tsx:77]: It was in case we didn't import path at all :sweat_smile:

Comment (kolibri753) [reply apps/frontend/src/pages/auth/auth.tsx:77]: Oh.. I was totally thinking the same thing and asked about it earlier, but unfortunately, you didn't see it then. Okay, I'll just revert it to how it was :black_cat:

Comment (liza-veis) [reply apps/frontend/src/pages/auth/auth.tsx:77]: oh sorry, probably missed that question

Comment (kolibri753) [reply apps/frontend/src/pages/auth/auth.tsx]: no worries :smile_cat: :

File: apps/frontend/src/pages/auth/auth.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { useNavigate } from "react-router-dom";
 
+import logoSrc from "~/assets/images/icons/logo.svg";
```
Comment (kolibri753) [inline apps/frontend/src/pages/auth/auth.tsx:3]: not sure about the path here

**Review by liza-veis — APPROVED** [review]

**Pull request merged**